cwt/draft-ietf-ace-cbor-web-token.xml

```diff
@@ -427,12 +425,16 @@
               Verify that the CWT is a valid CBOR object.
             </t>
             <t>
-              If the object begins with the CWT CBOR tag, remove it
-	      and verify that one of the COSE CBOR tags follows it.
-	    </t>
-	    <t>
-	      If the object is tagged with one of the COSE CBOR tags, remove it
-	      and verify that it corresponds to the structure of the following COSE object.
+              If the object begins with the CWT CBOR tag, remove it and
+              continue verification.
```

Comment:
My understanding was that we only forced the use COSE CBOR tags on the inner object when objects was nested. The CWT tag will always be on the outer most object.